NEWS.md

```diff
@@ -1,5 +1,9 @@
 # pkgdown (development version)
 
+* Vignettes/articles that use `<h1>` as top-level heading automatically have
```

Comment:
I now wonder whether it's the right decision (here and for changelogs). The page title h1 should maybe not count in the hierarchy especially as it's styled differently anyway? In Google docs there's title, subtitle and then heading 1, heading 2 etc.